features/steps/insights_sha_extractor.py

```diff
@@ -105,7 +106,10 @@ def check_b64_decode(context):
     assert message_in_buffer(
         expected_msg,
         context.sha_extractor.stdout
-    ), "message was not consumed"
+    ), "b64_identity was not extracted"
+
+    # kill the process so we have one consumer at a time
+    context.sha_extractor.terminate()
```

Comment:
you need to wait for termination
also better to terminate + kill, just in case

Reply:
it allows to donwload from here https://github.com/RedHatInsights/insights-behavioral-spec/pull/317/files#diff-24bf4bd473ed35b9af4a0ad39287206996c0da6abd5d435a6ff4b376651e45eaR12 so I can use the mock